compiler/luci/pass/src/FuseBatchNormWithConvPass.cpp

```diff
@@ -195,6 +197,12 @@ bool fused_batch_norm_with_conv(luci::CircleAdd *add)
   fused_conv->dilation()->h(conv->dilation()->h());
   fused_conv->dilation()->w(conv->dilation()->w());
 
+  auto pdata = std::make_unique<luci::ProfilingData>();
+  pdata->add_origins(conv->annot<luci::ProfilingData>()->origins());
+  pdata->add_origins(mul->annot<luci::ProfilingData>()->origins());
+  pdata->add_origins(add->annot<luci::ProfilingData>()->origins());
+  fused_conv->annot(std::move(pdata));
```

Comment:
Looks like this PR don't have part to assign `pdata->node_id`. I guess that would be done in export stage?

Reply:
Yes, operator index(node_id) of newly created nodes can be got only at export step :D
The only thing we can do in here is recording origins :)